pinotdb/db.py

```diff
@@ -47,6 +46,7 @@ def connect_async(*args, **kwargs):
 def check_closed(f):
     """Decorator that checks if connection/cursor is closed."""
 
+    # TODO: make sure the decorated function is properly wrapped.
```

Comment:
could you be more specific? what's the goal of this TODO?

Reply:
We should use functools.wraps here so that the decorated function doesn't lose its name. See here: https://docs.python.org/3/library/functools.html#functools.wraps - this is important to use when decorating a function, because, if not, then it might end up messing up with tools that depend on code inspection, like NewRelic agent etc.

Reply:
Fixed!

Reply:
Thanks for the clarification!